### ISSUE TITLE
Should I run `HEAD`?

### ISSUE THREAD

**Issue by ocharles** [tweag/rules_haskell#1622]
I'm evaluating `rules_haskell`, and I see a lot of active development but the last release was over a year ago. Should I run `HEAD`, or should I still use the latest release?

**Comment by aherrmann** [comment]
Thanks for reaching out. I would recommend using `HEAD` at this point.

**Comment by ocharles** [comment]
Ok, thanks!

**Comment by avdv** [comment]
Hi. :smiley: 

Just a quick follow-up question: what is the current minimum bazel version supported by HEAD?

Asking because I was trying to use bazel 3.7.2 with commit d3d8bede5597c4ffc373f9742a6a733c728505b2 and got this:
```
ERROR: /home/claudio/.cache/bazel/_bazel_claudio/a0b0c5e02d9eb86a96afefb39001e2b9/external/rules_haskell/haskell/cabal.bzl:1955:9: name 'json' is not defined
ERROR: /home/claudio/.cache/bazel/_bazel_claudio/a0b0c5e02d9eb86a96afefb39001e2b9/external/rules_haskell/haskell/cabal.bzl:2116:22: name 'json' is not defined
ERROR: error loading package '': Extension 'haskell/cabal.bzl' has errors
```
AFAICS, the `json` module is only available with bazel +4.0.0. Or is it possible to add this module for older bazel versions somehow?

**Comment by aherrmann** [comment]
Hi @avdv, good catch. Yes, `json` is only available in Bazel +4.0.0. So, the bounds should be bumped accordingly. Do you need to use rules_haskell with an older Bazel version?

**Comment by avdv** [comment]
Thanks for confirming, @aherrmann . I have no need to stay at bazel 3.x, and I just upgraded to the bazel_4 package without much ado. :+1:
